systemtests/CMakeLists.txt

```diff
@@ -84,20 +84,19 @@ function(configurefilestosystemtest srcbasedir dirname globexpression
                                                                # with .ini files
     string(REPLACE "${srcbasedir}/${srcdirname}" "" TARGET_FILE ${TARGET_FILE})
     get_filename_component(DIR_NAME ${TARGET_FILE} DIRECTORY)
-    if(DIR_NAME MATCHES "python-modules$")
+
+    if(${CURRENT_FILE} MATCHES ".in$")
+      configure_file(
```

Comment:
does `configure_file()` create the required directories? Otherwise you might want to move lines 94-96 above the `if()`.

Reply:
Good point. configure_file does not create directories but that was no problem until now. However, I move the lines before if() as suggested to be sure.